docs/common.py

```diff
@@ -92,12 +92,6 @@
 
 .. |release_source_tarball| replace:: %(url_github_archive)s/geomesa_2.11-%(release_last)s.tar.gz
 
-.. |release_kafka08_plugin| replace:: `geomesa-kafka-08-gs-plugin-2.11-%(release_last)s.tar.gz <%(url_locationtech_release)s/geomesa-kafka-08-gs-plugin_2.11/%(release_last)s/geomesa-kafka-08-gs-plugin_2.11-%(release)s-install.tar.gz>`__
-
-.. |release_kafka09_plugin| replace:: `geomesa-kafka-09-gs-plugin-2.11-%(release_last)s.tar.gz <%(url_locationtech_release)s/geomesa-kafka-09-gs-plugin_2.11/%(release_last)s/geomesa-kafka-09-gs-plugin_2.11-%(release)s-install.tar.gz>`__
-
-.. |release_kafka10_plugin| replace:: `geomesa-kafka-10-gs-plugin-2.11-%(release_last)s.tar.gz <%(url_locationtech_release)s/geomesa-kafka-10-gs-plugin_2.11/%(release_last)s/geomesa-kafka-10-gs-plugin_2.11-%(release)s-install.tar.gz>`__
```

Comment:
Is there a Kafka release bundle?

Reply:
there is, but it's not broken out by kafka version anymore - these were left over from 1.x, but aren't actually used anywhere.